content/i18n/en/docs/language/detailed-rules/typecobol-functions-syntax.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Functions
+---
+
+# Detailled rules of procedures and functions syntax 
+
+#### Table of Contents
+This page describes many rules regarding function or procedure declaration/call.
+All these rules begin with **TCRFUN\_**.
+* [Grammar reference](#grammar)
```

Comment:
Links won't work with the removal of <a> tags

Reply:
Yes, I noticed that, this is strange because, it does not accept the syntax <a../> it accepts <a...></a>

Reply:
Yeah, this is because Github was implementing a uncommon use of a, which is false HTML5 spoken. a tags are made to have something inside. Here we can write nothing, but we need to write an opening **and** a closing tag.

Reply:
Nice to read your confirmation.

Reply:
:tada: